nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/prometheusutil/AbstractMetricsRegistry.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.nifi.prometheus.util;
+package org.apache.nifi.prometheusutil;
```

Comment:
This change seems untentional, should it be reverted?

Reply:
Yeah the new package name seems fine to me.  I didnt see value in it being further lower.

The stuff used to be in extension-utils and is no longer needed plus i'd like to see nothing in the framework folders know anything about extension paths so this was a win.